app/assets/stylesheets/base/_typography.scss

```diff
@@ -30,4 +30,9 @@ hr {
   margin: 1em 0;
 }
 
+.italic {
```

Comment:
This style is used only in the context of a specific form, where it's not possible ATM to add criteria.

1. In order to be semantic, its naming must follow purpose : what exactly does it communicate to the user ? 
2. If it's being used only in the context of a [proun](https://github.com/sgmap/boussole/blob/master/app/assets/stylesheets/prouns/_all.scss), which are modulable pieces of html/css, it should be scoped to it and not be defined globally.
3. font-size must be defined using ```em```, otherwise it breaks responsiveness.